hack/tools/haproxy/README.md

```diff
@@ -208,3 +209,40 @@ Or, to produce a list of URLs for the same image files (and their checksums), th
 ```shell
 gsutil ls gs://capv-images/extra/haproxy/release/*/*.{ova,sha256} | sed 's~^gs://~http://storage.googleapis.com/~'
 ```
+
+## Template
+
+In production the haproxy loadbalancer is deployed as an template.
+
+### Template Requirements
+
+Building the template requires:
+
+* VMware vSphere >= 6.5
```

Comment:
we currently do not test against anything less than vsphere 6.7u3. I'd suggest bumping this to 6.7u3

Reply:
Ok, for the moment I'm only able to test with vsphere 6.5,  I should be soon be able to test with vSphere 6.7u3 and 6.5, do you prefer me to bump it ?